src/matlab2tikz.m

```diff
@@ -4562,9 +4562,17 @@ function warnFacetedInterp(m2t)
         end
     end
 
-    % Merge them together in axisOptions.
+   % Merge them together in axisOptions.
     axisOptions = opts_add(axisOptions, strtrim(['colorbar ', cbarTemplate]));
 
+   
+    %color map ticks
+    [m2t, options] = getAxisOptions(m2t, handle, 'y');
```

Comment:
Here you can just pass cbarStyleOptions directly.

Then there is no need to merge below.